### DISCUSSION TITLE
Facing issues with FHIR STU3ToR4 liquid templates

### DISCUSSION THREAD

**Discussion by shashisonga** [microsoft/FHIR-Converter#573] [General]
Hi All,
I am using the FHIR STU3ToR4 templates. Tried this using the FHIR examples for Medication/Encounter resources. R3 examples tried are from https://hl7.org/fhir/STU3/medication-examples.html. But the resultant R4 is not matching to the FHIR R4 from https://hl7.org/fhir/R4/medication.html. 
Any one faced these issues. Any suggestions ?

**Comment by dustinburson** [comment]
Hello, the templates provide are examples and should be modified meet your specific needs.  The templates in question aren't validated against STU3 samples.  That said, if there are instances where you feel the default templates are incorrect please provide specific examples (i.e. with this input STU3 resource I am not seeing X expected elements in R4) and we will look at how they might be improved.  Thanks!
